src/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM  from 'react-dom';
+import CustomComponent from './components/CustomComponent';
+import CustomPureComponent from './components/CustomPureComponent';
+
+const elemStyle = {
+    width: '150px',
+    backgroundColor: 'blue',
+    color: 'white',
+    padding: '15px',
+    margin: '15px'
+}
```

Comment:
I hope such styles object will be only here in example :)

Reply:
Yes, only here ))